src/docs/admin/references/NodeSourcePolicyReference.adoc

```diff
@@ -332,7 +337,7 @@ and so on ...
 The load factor accepts decimal values to allow more precise control (1.5, etc.)
 
 -   *initDelay* - Delay in milliseconds to initialize the infrastructure (eg. in a azure cloud infrastructure this must
-cover the creation of Azure's initial resources such as network, storage account, etc.).
+cover the creation of Azure's initial resources such as network, storage account, etc.). This parameter is also used as trigger of the *best-effort scale-up strategy* described above.
```

Comment:
This parameter is also used as **a** trigger of the *best-effort scale-up strategy* described above.